packages/alfa-map/test/map.spec.ts

```diff
@@ -49,7 +49,7 @@ test("#apply() applies a map of functions to each corresponding value of a map",
     [
       ...map.apply(
         Map.of(
-          ["foo", (x) => x + 1],
+          ["foo", (x: number) => x + 1],
```

Comment:
Not sure why these now need some typing. I assume it is also due to the changes around `unknown` in TS 4.8.